instrumentation/mysql2/test/opentelemetry/instrumentation/mysql2/instrumentation_test.rb

```diff
@@ -133,10 +133,47 @@
       assert(!span.events.first.attributes['exception.stacktrace'].nil?)
     end
 
+    describe 'when db_statement set as obfuscate' do
```

Comment:
I think we may also want at least one more check here where for some reason someone sets both the legacy attribute and the new one. _Just in case_

Reply:
Would need to be able to tell then when something is a default, vs user chose default value. I might have to remove https://github.com/open-telemetry/opentelemetry-ruby/pull/894/files#diff-335a880214cc43314c9840b69feddb39868a51db2dda1ffb126094486f6ec39bL23 the default setting and handle it inside the `install`, to be able to differentiate between the two.